dashboards/Experimental/Environments_Overview.json

```diff
@@ -1421,6 +1700,7 @@
       },
       "yaxes": [
         {
+          "$$hashKey": "object:73",
```

Comment:
We need to remove those $$hasKey entries - they are redundant and not good for maintenance.